app/src/layouts/map/map.vue

```diff
@@ -227,6 +227,11 @@ export default defineComponent({
 	background-color: var(--background-page);
 	border-radius: var(--border-radius);
 	box-shadow: var(--card-shadow);
+	pointer-events: none;
+}
+
+.v-info > ::v-deep(.v-button) {
```

Comment:
We use the pseudo-class syntax everywhere else:

```suggestion
.v-info > :deep(.v-button) {
```

This should also make the change to `.stylelintrc.json` not necessary.